classes/metric/manager.php

```diff
@@ -74,6 +74,13 @@ public static function get_frequency_labels(): array {
         ];
     }
 
+    /** @var int[] Default settings for builtin metric frequencies. */
```

Comment:
I don't think the type correctly reflects the content. I don't remember the correct one specifically, but it might be something like `int[string]` instead?

Reply:
done